content/systems/filecoin_vm/sysactors/cron_actor.md

```diff
@@ -10,4 +10,25 @@ dashboardTests: 0
 # CronActor
 ---
 
-{{<embed src="/modules/actors/builtin/cron/cron_actor.go"  lang="go">}}
+The `CronActor` is responsible for sending messages to other registered actors at the end of every epoch. It interfaces with the Storage Power Actor on `OnEpochTickEnd` to check:
+- `CronEventPreCommitExpiry`: if the miner has submitted their `ProveCommit` proofs,
+- `CronEventProvingPeriod`: the proving period identified,
+- `CronEventWorkerKeyChange`: if a miner has submitted a worker key address change.
```

Comment:
IMO, the cron actor should not be defined in terms of the other actors that use it, but the service that it provides.

Reply:
I would just say that, built in to the genesis state, the CronActor's dispatch table invokes the StoragePowerActor and StorageMarket actor for them to maintain internal state and process deferred events. It could in principle invoke other actors after a network upgrade.

There's no " `OnEpochTickEnd` call the `InitActor`"